cmd/tsconnect/wasm/wasm_js.go

```diff
@@ -195,6 +204,22 @@ func newIPN(jsConfig js.Value) map[string]any {
 			url := args[0].String()
 			return jsIPN.fetch(url)
 		}),
+		"tcp": js.FuncOf(func(this js.Value, args []js.Value) any {
+			if len(args) != 3 && len(args) != 4 {
+				log.Printf("Usage: tcp(hostname, port, readCallback, connectTimeoutSeconds)")
+				return nil
+			}
+			connectTimeoutSeconds := 0
+			if len(args) == 4 && args[3].Type() == js.TypeNumber {
+				connectTimeoutSeconds = args[3].Int()
+			}
```

Comment:
let's use float64 (a javascript number) instead of int. Then we can express "0.3" for 300 ms, etc.